docs/CONTINUOUS_DEPLOYMENT_GUIDE.md

```diff
@@ -0,0 +1,38 @@
+## Continuous Deployment Guide
+
+As a quick start, you can utilize the available bash script which is capable of deploying both modules to a 
+Google Cloud Project. It requires to have:
+
+- [gcloud sdk](https://cloud.google.com/sdk/docs/install)
+- [docker](https://docs.docker.com/get-docker/)
+- [helm](https://helm.sh/docs/intro/install/)
+- [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+- an existing Kubernetes cluster in [Google Kubernetes Engine](https://cloud.google.com/kubernetes-engine/docs/quickstart)
+
+
+### Prerequisites:
+1. Configure both module specific configurations in yaml files that resides under `k8s` folders.
```

Comment:
To configure those vars, you need to go to k8s-charts repo. So give a link to that repo.